source/actors/player/PlayerHealth.hx

```diff
@@ -0,0 +1,141 @@
+package actors.player;
+
+import flixel.group.FlxGroup;
+
+class PlayerHealth {
+
+    public var healthGraphicGroup(default, null):FlxTypedGroup<HeartContainer>;
+    public var maxHealth(default, null):Int = 12;
```

Comment:
Maybe make the maxHealth a parameter that can be passed into the constructor, so whatever's setting it up is responsible for dictating starting health. Avoids situations where we accidentally don't provide anything, since the class requires it.